packages/mdc-animation/README.md

```diff
@@ -1,6 +1,6 @@
 # mdc-animation
```

Comment:
I think we should add a section in the README which documents the JS utility functions